policies/documentation-policy.md

```diff
@@ -0,0 +1,122 @@
+# OpenSSL documentation policy
+
+This document describes the code documentation and commenting
+requirements for the OpenSSL project.
+
+The project's documentation is all about making the libraries and tools
+more accessible to our users and making the code more maintainable.
+This policy applies to new submissions, existing code is below par and will
+be gradually improved.
```

Comment:
I think our policy should not say that the existing code is below par. It also seems to say that existing documentation doesn't be fixed, it only applies to new code. But I expect it to say when making a non-trivial change to the code, the documentation should get checked and updated if needed.